openid-connect-4-credential-issuance-1_0.md

```diff
@@ -347,15 +436,71 @@ Below is a non-normative example of a `claims` parameter including the `credenti
 }
 ```
 
+### Pushed Authorization Request
+
+This step is OPTIONAL. In case of complex flows or when the size of the request is large due to the query language or encrypting, clients are RECOMMENDED to use pushed uthorization requests.
```

Comment:
A PAR must be send before the actual authz reqzest.

Reply:
Yes, but to have an example of the parameters defined in the authorization req section, I put it after for now.